### PR TITLE
Add structure for generators

### PR DESCRIPTION
**Pull request by shal**

### CODE REVIEW

File: kubelogic.gemspec [hunk 1]
```diff
@@ -23,4 +23,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.16"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "rails", "~> 3.0"
```
Comment (dmk) [inline kubelogic.gemspec]: Why development dependency?
And why rails 3.0?

Comment (shal) [reply kubelogic.gemspec]: What rails version should be? I think greater that 3.0

File: lib/generators/kubelogic/config_generator.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
```
Comment (mod) [inline lib/generators/kubelogic/config_generator.rb:1]: I believe it should be lib/kubelogic/generators/

**Review by mod — CHANGES_REQUESTED** [review]
congrats very good start
but what about seeding configs to DB ?

File: lib/generators/kubelogic/config_generator.rb (hunk 2 above)
Comment (shal) [reply lib/generators/kubelogic/config_generator.rb:1]: @mod In this case I got error like 

```
rails g kubelogic:config          
Could not find generator 'kubelogic:config'. Maybe you meant 'kubelogic:config', 'kaminari:config' or 'kubelogic'
Run `rails generate --help` for more options.
```

**Comment by shal** [comment]
@mod Next step will we work on generating configs to DB. This PR is just adding structure for rails generators

**Pull request merged**